packages/react/package.json

```diff
@@ -202,7 +204,7 @@
     "styled-components": "5.3.11",
     "terser": "5.36.0",
     "ts-toolbelt": "9.6.0",
-    "tsx": "4.7.0",
+    "tsx": "4.20.3",
```

Comment:
Older version of `tsx` was having issues running the esm build of `markdownlint`